client/components/dropdown/index.js

```diff
@@ -15,6 +15,7 @@ const Dropdown = ( { id, valuesMap, title, description, value, updateValue, erro
 				name={ id }
 				value={ value }
 				onChange={ ( event ) => updateValue( event.target.value ) }
+				disabled={ Boolean( disabled ) }
```

Comment:
If the prop type is required to be bool, why do we need Boolean here? Would having getDefaultProps instead be the more common pattern for React?